pypesto/petab/importer.py

```diff
@@ -417,6 +424,31 @@ def create_objective(
             # FIXME: currently not supported with hierarchical
             kwargs['guess_steadystate'] = False
 
+        if self._ordinal:
+            # TODO add constants to C
+            if 'inner_problem_method' in kwargs:
+                inner_problem_method = kwargs.pop('inner_problem_method')
+            else:
+                inner_problem_method = None
```

Comment:
```suggestion
            inner_problem_method = kwargs.get('inner_problem_method', None)
```

Reply:
Think get will not work, as described up.
But I put this: 
            inner_problem_method = kwargs.pop(INNER_PROBLEM_METHOD, None)